specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2024-07-01-preview/elasticsan.json

```diff
@@ -931,6 +976,21 @@
               "modelAsString": true
             },
             "description": "Optional, used to delete volume if active sessions present. Allowed value are only true or false. Default value is false."
+          },
+          {
+            "name": "purge",
+            "description": "Optional, used to purge soft deleted volumes if set to true.",
```

Comment:
Does this mean it will be permanent deleted?
What's the default value? 
Please update the description to include these info and make it more clear.

Besides that, if this is for permanent delete:
1. Permanent deleted blob parameter is hidden in storage SDK/PSH/CLI. Should we also hide this parameter in PSH/CLI/SDK?
2. Why not use aligned parameter name/type as [blob permanent delete](https://learn.microsoft.com/en-us/rest/api/storageservices/delete-blob?tabs=microsoft-entra-id#:~:text=Storage%20operations.-,deletetype,-Optional%2C%20version%202020)?

Reply:
Duplicate comment